src/features/course-player/CoursePlayer.jsx

```diff
@@ -168,7 +166,7 @@ const CoursePlayer = (props) => {
   return (
     <div className="course-player-main-content">
       {/* Avoud crashes with invalid course or sequence states */}
-      {!(courseId !== (routeCourseId || null) || sequenceId !== (routeSequenceId || null)) && (
+      {/* {!(courseId !== (routeCourseId || null) || sequenceId !== (routeSequenceId || null)) && ( */}
```

Comment:
This is needed otherwise we can get crashes, or did you fix it otherwise?

Reply:
That i wanted to discuss with you, i need to see what is the issue.